drivers/include/enc28j60.h

```diff
@@ -7,7 +7,7 @@
  */
 
 /**
- * @defgroup    driver_enc28j60 ENC28J60
+ * @defgroup    drivers_enc28j60 ENC28J60
```

Comment:
@gebart I piggy-backed https://github.com/RIOT-OS/RIOT/pull/6559/files#r99790112 here, since it is topically related to this PR